src/recordlinker/main.py

```diff
@@ -201,3 +210,13 @@ async def link_record(
             "updated_bundle": input_bundle,
             "message": f"Could not connect to database: {err}",
         }
+
+@app.get("/algorithms")
+async def get_algorithms() -> GetAlgorithmsResponse:
```

Comment:
What do you think about renaming this to be get_algorithm_labels?  Technically we're not returning the whole algorithm, just the available labels.